code/builders/builder-webpack5/package.json

```diff
@@ -64,7 +64,7 @@
   },
   "dependencies": {
     "@storybook/core-webpack": "workspace:*",
-    "@types/node": "^18.0.0",
+    "@types/node": "^22.0.0",
```

Comment:
This change force-upgraded `@types/node` in our app repo to 22.x while we're using Node 20.x and don't want typings for a newer Node version (remember, these things are global). This effectively made Node 22.x the minimum required version of Node for Storybook with Webpack if you're using TypeScript.